### PR TITLE
enable select issueType closes #4

### LINKED ISSUE #4: enable selection of issueType
issues & pull requests
only issues
only pull request

### PR DESCRIPTION
**Pull request by atsfour**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -209,19 +238,23 @@
...
+    }
 
-   if (inlineApp.urlsNum == 1) {
-     $('#hideRepoNames').attr('checked', 'checked');
-     inlineApp.toggleVisibility();
-   }
+    if (inlineApp.urlsNum == 1) {
+      $('#hideRepoNames').attr('checked', 'checked');
+      inlineApp.toggleVisibility();
+    }
```
Comment (atsfour) [inline index.html:264]: fixed only indents.

File: index.html [hunk 2]
```diff
@@ -147,10 +149,18 @@
...
+    var pushDatum = function(obj, key, data) {
+      if (!obj[key]) {
+        obj[key] = [];
+      }
+      obj[key].push(data);
+    }
+    if (d.pull_request) {
+      d.isPr = true;
+    } else {
+      d.isPr = false;
```
Comment (hiraiva) [inline index.html]: It might be better to use ternary operator.

File: index.html [hunk 3]
```diff
@@ -167,13 +177,32 @@
...
+      }
+      return res;
+    }
+    if (inlineApp.issueType === 'all') {
+      data = inlineApp.data;
+    } else if (inlineApp.issueType === 'onlyIssues') {
+      data = filterObj(inlineApp.data, function(d){ return !d.isPr; } )
+    } else if (inlineApp.issueType === 'onlyPRs') {
+      data = filterObj(inlineApp.data, function(d){ return d.isPr; } )
+    }
```
Comment (hiraiva) [inline index.html:213]: `switch` syntax be able to use in JS too.

File: index.html [hunk 4]
```diff
@@ -167,13 +177,32 @@
...
+  },
+  renderTable: function(data, table) {
+    var thead = table.find('thead');
+    var tbody = table.find('tbody');
+    var rawKeys = Object.keys(data);
+    var max = 0; // most assigned person's issues
     var all = 0;
     var assigned = 0;
-    var assignees = [];
+    var assignees = []; // assignees sorted by name
```
Comment (ushjp) [inline index.html]: [nits] sorted except for NOT_ASSIGNED

**Comment by hiraiva** [comment]
LGTM

**Comment by ushjp** [comment]
LGTM :+1:

**Pull request merged**